packages/ethers-lib/src/contracts/SmartWalletFactory/v1.0.0/SmartWalletProxyFactoryEthersContract.ts

```diff
@@ -16,7 +16,7 @@ class SmartWalletFactoryContractEthers implements SmartWalletFactoryContract {
   }
 
   async isWalletExist(wallet: string): Promise<boolean> {
-    const doesExist = await this.contract.isWalletExist(wallet)
+    const doesExist = await this.contract.isAccountExist(wallet)
```

Comment:
we will be possibly removing this... watchout for upcoming PR's on c4-remediations branch